package.json

```diff
@@ -5,7 +5,12 @@
   "description": "Foreman isn't really a node module, these are just dependencies needed to build the webpack bundle. 'dependencies' are the asset libraries in use and 'devDependencies' are used for the build process.",
   "private": true,
   "devDependencies": {
-    "@kadira/storybook": "^2.5.2",
+    "@storybook/addon-actions": "^3.1.2",
+    "@storybook/addon-info": "^3.1.4",
+    "@storybook/addon-knobs": "3.1.2",
```

Comment:
Is there a reason this one is locked to a specific version and the other addons are not? 
Also, do we actually use any of the addons other then actions anywhere?